### PR TITLE
Feature responsive profile

### PR DESCRIPTION
**Pull request by bleachjade**
I've added messages when you joined, left, or deleted an event and also made the profile page to responsive on all devices.

### CODE REVIEW

**Comment by BenZacs** [comment]
the alert message should be "joined" and "deleted".

File: Kvent/static/css/profile.css [hunk 1]
```diff
@@ -1,66 +1,111 @@
...
+    padding: 0 0 0 1em;
     text-decoration: none;
-    font-size: 25px;
+    color: white;
+}
+.nav-bar ul {
+    display: flex;
 }
-
-
```
Comment (JirawadeeSampusri) [inline Kvent/static/css/profile.css:31]: please delete space

File: Kvent/templates/kvent/profile.html [hunk 2]
```diff
@@ -20,41 +20,31 @@
     <ul>
         {% if user.is_authenticated %}
         <li><a href="{% url 'event-history' username=request.user.username %}" id="event-logo">Your events</a></li>
+        <li><a href="{% url 'profile' %}" id="event-logo">Profile</a></li>
         {% endif%}
     </ul>
     </nav>
```
Comment (JirawadeeSampusri) [inline Kvent/templates/kvent/profile.html:27]: please delete space

File: Kvent/templates/kvent/index.html [hunk 3]
```diff
@@ -12,6 +12,15 @@
 </head>
```
Comment (JirawadeeSampusri) [inline Kvent/templates/kvent/index.html:13]: please delete space

**Pull request merged**